src/files/pyfingerprint/pyfingerprint.py

```diff
@@ -639,16 +661,21 @@ def getTemplateCount(self):
         else:
             raise Exception('Unknown error '+ hex(receivedPacketPayload[0]))
 
-    def readImage(self):
+    def readImage(self, controlLighting=True):
```

Comment:
Documentation of parameter is missing ;)

Reply:
Done

Reply:
I don't like the naming, maybe "enableSensorLed" or something like this

Reply:
`enableSensorLed` isn't necessary accurate. if the LED has manually been switched on, then the module will automatically switch it off after attempting the read. Would `automaticLighting` be better?

Reply:
Okay sorry i didn't read your added code documentation. Yeah you are right, i think `automaticLedControl = True` would be accurate for naming.

Reply:
@cyberjacob please rename parameter to `automaticLedControl` and we can approve and close your pull request ;)